tests/test_basic_1.py

```diff
@@ -594,112 +574,139 @@ def test_query(db):
-    assert type(white_car_id) == int
-    assert result == [{'color': 'white', 'make': 'Toyota', 'neo4j_labels': ['car'], 'neo4j_id': white_car_id}]
-
-    q = "MATCH (x) RETURN x"
-    result = db.query_expanded(q, flatten=True)
-    assert result == [{'color': 'white', 'make': 'Toyota', 'neo4j_labels': ['car'], 'neo4j_id': white_car_id}]
-
-    # Create and return 2 more nodes at once
-    q = '''CREATE (b:boat {number_masts: 2, year:2003}),
-                  (c:car {color: "blue"})
-           RETURN b, c
-        '''
-    result = db.query_expanded(q, flatten=True)
-    for node_dict in result:
-        if node_dict['neo4j_labels'] == ['boat']:
-            boat_id = node_dict['neo4j_id']
-        else:
-            blue_car_id = node_dict['neo4j_id']
-
-    assert result == [{'number_masts': 2, 'year': 2003, 'neo4j_labels': ['boat'], 'neo4j_id': boat_id},
-                      {'color': 'blue', 'neo4j_labels': ['car'], 'neo4j_id': blue_car_id}]
+def test_query_neo4jResult(db):
+    result = db.query("RETURN 1", return_type="neo4j.Result")
+    assert isinstance(result, neo4j.Result)
 
-    # Retrieve all 3 nodes at once
-    q = "MATCH (x) RETURN x"
-    result = db.query_expanded(q, flatten=True)
-    assert result == [{'color': 'white', 'make': 'Toyota', 'neo4j_labels': ['car'], 'neo4j_id': white_car_id},
-                      {'number_masts': 2, 'year': 2003, 'neo4j_labels': ['boat'], 'neo4j_id': boat_id},
-                      {'color': 'blue', 'neo4j_labels': ['car'], 'neo4j_id': blue_car_id}]
-
-    q = "MATCH (b:boat), (c:car) RETURN b, c"
-    result = db.query_expanded(q, flatten=True)
-    assert result == [{'number_masts': 2, 'year': 2003, 'neo4j_id': boat_id, 'neo4j_labels': ['boat']},
-                      {'color': 'white', 'make': 'Toyota', 'neo4j_id': white_car_id, 'neo4j_labels': ['car']},
-                      {'number_masts': 2, 'year': 2003, 'neo4j_id': boat_id, 'neo4j_labels': ['boat']},
-                      {'color': 'blue', 'neo4j_id': blue_car_id, 'neo4j_labels': ['car']}]
-
-    result = db.query_expanded(q, flatten=False)    # Same as above, but without flattening
-    assert result == [
-                        [{'number_masts': 2, 'year': 2003, 'neo4j_id': boat_id, 'neo4j_labels': ['boat']},
-                         {'color': 'white', 'make': 'Toyota', 'neo4j_id': white_car_id, 'neo4j_labels': ['car']}
-                        ],
-                        [{'number_masts': 2, 'year': 2003, 'neo4j_id': boat_id, 'neo4j_labels': ['boat']},
-                         {'color': 'blue', 'neo4j_id': blue_car_id, 'neo4j_labels': ['car']}
-                        ]
-                      ]
 
+def test_query_pd(db):
+    result = db.query("UNWIND [{`key 1`: 1, `key 2`: 2}, {`key 1`: 3, `key 2`: 4}] as map RETURN map",
+                      return_type="pd")
+    assert isinstance(result, pd.DataFrame)
+    assert result.equals(pd.DataFrame([{'key 1': 1, 'key 2': 2}, {'key 1': 3, 'key 2': 4}]))
```

Comment:
The map in the cypher query is not ordered and actually returns 
![image](https://user-images.githubusercontent.com/96111417/166673846-ec825fb1-001c-46f0-aed6-2ef81f978521.png)

If it does not matter what the order of the columns on return are, we could use the pandas.testing assert_frame_equals with the parameter check_like=True to assert the equality of the dataframes regardless of the order of either indexes.

E.g. `assert_frame_equal(result, pd.DataFrame([{'key 1': 1, 'key 2': 2}, {'key 1': 3, 'key 2': 4}]), check_like=True)`